src/startupscriptgenerator/common/outputHelper.go

```diff
@@ -0,0 +1,68 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func ExtractZippedOutput(appFolder string) {
+	zipFileName := "oryx_output.tar.gz"
+	scriptPath := "/tmp/extractZippedOutput.sh"
+	scriptBuilder := strings.Builder{}
+	scriptBuilder.WriteString("#!/bin/sh\n")
```

Comment:
are those standalone scripts? Ideally we should have a single script that can run the app

Reply:
Fixed this.